compose-app/src/main/java/com/mapbox/maps/compose/testapp/data/model/ExampleEntry.kt

```diff
@@ -0,0 +1,37 @@
+package com.mapbox.maps.compose.testapp.data.model
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+public sealed class ExampleEntry
+
+@Parcelize
+public data class ExampleCategory(
+  val category: String
+) : ExampleEntry(), Parcelable
+
+@Parcelize
+public data class SpecificExample(
+  val name: String,
+  private val label: String? = null,
+  private val description: String? = null,
+  val category: String
+) : ExampleEntry(), Parcelable {
+
+  val simpleName: String
+    get() = buildSimpleName()
+
+  private fun buildSimpleName(): String {
+    val split =
+      name.split("\\.".toRegex()).dropLastWhile { it.isEmpty() }.toTypedArray()
+    return split[split.size - 1]
```

Comment:
should we check empty name?

Reply:
The name comes from `android.name ` of the activity in Manifest, it will not be empty, but we can add a check here.

Reply:
updated